pages.th/linux/ac.md

```diff
@@ -0,0 +1,24 @@
+# ac
+
+> แสดงสถิติที่ผู้ใช้งานได้เชื่อมต่อเข้ามาในระบบเป็นเวลานานเท่าไหร่
+> ข้อมูลเพิ่มเติม: <https://www.gnu.org/software/acct/manual/accounting.html#ac>.
+
+- แสดงสถิติเวลาที่ผู้ใช้ปัจจุบันได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง
+
+`ac`
+
+- แสดงสถิติเวลาที่ผู้ใช้ทุกคนได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง แจกแจงตามผู้ใช้แต่ละคน
+
+`ac --individual-totals`
+
+- แสดงสถิติเวลาของผู้ใช้ที่ระบุไว้ได้เชื่อมต่อเข้ามาในระบบในหลักชั่วโมง
+
+`ac --individual-totals {{username}}`
```

Comment:
Please translate the placeholders, like `username`, for get this easier for non-English speakers.

Reply:
Yes, I just added the translation as suggested :).